solo-chains/runtime/dancelight/src/tests/common/xcm/expected_event_checker.rs

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+#[macro_export]
+macro_rules! assert_expected_events {
```

Comment:
Why is this here? Can't we use the one from polkadot-sdk?

Reply:
I am sure there was a good reason for bringing it. But we can try using the polkadot one